### PR TITLE
[plugin.video.dumpert] 1.1.9

### PR DESCRIPTION
**Pull request by skipmodea1**
### Description
v1.1.9 (2019-09-19) (thanks for the help NightFox89)
- MAJOR changes in the addon due to redesigned website
- updated logos
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

File: plugin.video.dumpert/resources/lib/dumpert_main.py [hunk 1]
```diff
@@ -52,50 +56,72 @@ def __init__(self):
...
-        # Toppers for a given date (from mobile site)
+        # Dag Toppers
+        #
+        parameters = {"action": "json",
+                      "plugin_category": LANGUAGE(30008),
+                      "period": DAY,
+                      "days_deducted_from_today": "0",
+                      "next_page_possible": "True"}
+        url = self.plugin_url + '?' + urllib.parse.urlencode(parameters)
+        list_item = xbmcgui.ListItem(LANGUAGE(30008), iconImage="DefaultFolder.png")
```
Comment (enen92) [inline plugin.video.dumpert/resources/lib/dumpert_main.py]: Remove kwargs

File: plugin.video.dumpert/resources/lib/dumpert_main.py [hunk 2]
```diff
@@ -42,8 +44,10 @@ def __init__(self):
...
         # Toppers
         #
-        parameters = {"action": "list", "plugin_category": LANGUAGE(30000),
-                      "url": "http://www.dumpert.nl/toppers/1/", "next_page_possible": "True"}
+        parameters = {"action": "json",
+                      "plugin_category": LANGUAGE(30000),
+                      "url": TOPPERS_URL,
+                      "next_page_possible": "True"}
         url = self.plugin_url + '?' + urllib.parse.urlencode(parameters)
         list_item = xbmcgui.ListItem(LANGUAGE(30000), iconImage="DefaultFolder.png")
```
Comment (enen92) [inline plugin.video.dumpert/resources/lib/dumpert_main.py]: Remove iconimage kwarg

File: plugin.video.dumpert/resources/lib/dumpert_json.py [hunk 3]
```diff
@@ -85,54 +197,98 @@ def getVideos(self):
...
+                is_folder = False
+                # Add refresh option to context menu
+                list_item.addContextMenuItems([('Refresh', 'Container.Refresh')])
+                # Add our item to the listing as a 3-element tuple.
+                listing.append((url, list_item, is_folder))
 
         # Next page entry
         if self.next_page_possible == 'True':
-            next_page = self.current_page + 1
             list_item = xbmcgui.ListItem(LANGUAGE(30503), thumbnailImage=os.path.join(IMAGES_PATH, 'next-page.png'))
```
Comment (enen92) [inline plugin.video.dumpert/resources/lib/dumpert_json.py]: Remove thumbnail image kwarg

File: plugin.video.dumpert/resources/lib/dumpert_json.py [hunk 4]
```diff
@@ -85,54 +197,98 @@ def getVideos(self):
...
-                    list_item.setArt({'thumb': thumbnail_url, 'icon': thumbnail_url,
-                                      'fanart': os.path.join(IMAGES_PATH, 'fanart-blur.jpg')})
-                    list_item.setProperty('IsPlayable', 'true')
-                    parameters = {"action": "play-file", "file": file}
-                    url = self.plugin_url + '?' + urllib.parse.urlencode(parameters)
-                    is_folder = False
-                    # Add refresh option to context menu
-                    list_item.addContextMenuItems([('Refresh', 'Container.Refresh')])
-                    # Add our item to the listing as a 3-element tuple.
-                    listing.append((url, list_item, is_folder))
```
Comment (enen92) [inline plugin.video.dumpert/resources/lib/dumpert_json.py]: Remove kwarg

File: plugin.video.dumpert/resources/lib/dumpert_main.py [hunk 5]
```diff
@@ -30,7 +30,9 @@ def __init__(self):
         #
         # Nieuw
         #
-        parameters = {"action": "list", "plugin_category": LANGUAGE(30001), "url": "http://www.dumpert.nl/1/",
+        parameters = {"action": "json",
+                      "plugin_category": LANGUAGE(30001),
```
Comment (enen92) [inline plugin.video.dumpert/resources/lib/dumpert_main.py]: Remove kwarg

File: plugin.video.dumpert/todo.txt [hunk 6]
```diff
@@ -1,9 +1 @@
-Maybe use the mobile feed? Gives some extra options:
-- Nieuw: http://dumpert.nl/mobile_api/json/hotshiz/0/
-- Classics: http://dumpert.nl/mobile_api/json/classics/0/
-- Top Dag: http://dumpert.nl/mobile_api/json/top5/dag/2016-03-19/0/
-- Top Week: http://dumpert.nl/mobile_api/json/top5/week/201611/0/
-- Top Maand: http://dumpert.nl/mobile_api/json/top5/maand/201603/0/
-
-Mobile feed misses:
-- youtube embeds
+﻿
```
Comment (enen92) [inline plugin.video.dumpert/todo.txt]: Remove from submission

File: plugin.video.dumpert/resources/lib/dumpert_main.py (hunk 2 above)
Comment (skipmodea1) [reply plugin.video.dumpert/resources/lib/dumpert_main.py]: Done

File: plugin.video.dumpert/resources/lib/dumpert_json.py (hunk 3 above)
Comment (skipmodea1) [reply plugin.video.dumpert/resources/lib/dumpert_json.py]: Done

File: plugin.video.dumpert/resources/lib/dumpert_json.py (hunk 4 above)
Comment (skipmodea1) [reply plugin.video.dumpert/resources/lib/dumpert_json.py]: Done

File: plugin.video.dumpert/resources/lib/dumpert_main.py (hunk 5 above)
Comment (skipmodea1) [reply plugin.video.dumpert/resources/lib/dumpert_main.py]: Done

File: plugin.video.dumpert/todo.txt (hunk 6 above)
Comment (skipmodea1) [reply plugin.video.dumpert/todo.txt]: Done

**Comment by enen92** [comment]
Thanks for adjusting

**Pull request merged**